tests/test_types.py

```diff
@@ -51,6 +51,30 @@ def test_second_positional(self):
         assert config.e == "f"
 
 
+class TestBytes:
+    @dataclass
+    class Config:
+        a: bytes = b"a"
+        b: bytes = field(default=b"b", metadata=dict(encoding="ascii"))
+        z: int = 12
+
+    def test_default(self):
+        config = parse(self.Config, [])
+        assert config.a == b"a"
+        assert config.b == b"b"
+        assert config.z == 12
+
+    def test_encoding(self):
+        config = parse(self.Config, ["--a", "ꀀ"])
```

Comment:
😂 I wiped my screen to get rid of the dust...

.. but it was part of the character